packages/now-cli/src/commands/dev/dev.ts

```diff
@@ -93,5 +93,5 @@ export default async function dev(
   process.once('SIGINT', () => devServer.stop());
   process.once('SIGTERM', () => devServer.stop());
 
-  await devServer.start(...listen);
+  await devServer.start(client, link.project, ...listen);
```

Comment:
I think it's better to pass these arguments to the constructor of  `DevServer` since they're used on `getNowConfig` which is called from other methods as well. (tho it looks working fine because of cache).